.github/workflows/auto-reviewer.yml

```diff
@@ -1,12 +1,11 @@
 name: 'Reviewer lottery'
 on:
   pull_request:
-    types: [opened, ready_for_review, reopened]
-    tags-ignore:
-      - pair-programmed
+    types: [labeled]
```

Comment:
I think this means that it will only run when a label is added, and not even when the PR is `opened`, the evidence being this PR itself